Tools/wasm/emscripten/__main__.py

```diff
@@ -336,6 +362,7 @@ def main():
     context = parser.parse_args()
 
     dispatch = {
+        "make-libffi": make_emscripten_libffi,
```

Comment:
Does the README need to be updated to include this step as part of the "long version" of how to build?